### PR TITLE
OSX main menu bar

### PR DESCRIPTION
**Pull request by TheSpiritXIII**
Adds a global menu bar for OS X instead of having an empty one. Includes common OS X shortcuts/menus, including preferences, about and quit, as well as a few window-based shortcuts.

This does not include some actions such as cut/copy/paste because they're a little bit harder to implement for OS X. It would require signals on each widget which is tedious work and will likely be a different PR.

### CODE REVIEW

**Comment by tux3** [comment]
Your PR and commit message don't match what you changed. Don't do that.

It looks like you tried to fix the INSTALL.md, that's good, but put that change in a separate and properly named commit. I'm sure that was a mistake, but be careful please. Don't just hide unrelated changes in.

**Comment by TheSpiritXIII** [comment]
The change you're talking about is what I told you about before where you merged my OS X install instructions too early. zetok told me to fix the readme format for OS X in the IRC channel to match the others. I fixed it and squashed it but I was a few minutes too late and it could no longer be merged properly.

**Comment by tux3** [comment]
@zetok stop telling people to squash things, it messes with the history.

**Comment by TheSpiritXIII** [comment]
Removed the INSTALL.md changes.

**Comment by TheSpiritXIII** [comment]
Don't merge yet. Just realized that the menu bar doesn't play nicely with the login screen (since it's a global menu bar).

EDIT: It's working now.

File: src/widget/widget.cpp [hunk 1]
```diff
@@ -342,6 +415,11 @@ Widget::~Widget()
     delete trayMenu;
     delete ui;
     instance = nullptr;
+
+#ifdef Q_OS_MAC
+    //Nexus::getInstance().globalMenuBar->clear();
+    //Nexus::getInstance().globalMenuBar->addMenu(Nexus::getInstance().windowMenu);
+#endif
 }
```
Comment (tux3) [inline src/widget/widget.cpp:420]: Can be removed?

**Pull request merged**